options.js

```diff
@@ -12,13 +12,7 @@ browser.storage.local.get(['yourName', 'messageTemplates'])
         if (yourName) {
             document.getElementById('yourName').value = yourName
         }
-        if (typeof messageTemplates === 'string') {
-            // For backwards compatibility
-            messageTemplates = JSON.parse(messageTemplates)
-            browser.storage.local.set({
-                messageTemplates
-            })
-        }
```

Comment:
this was a backward compatibility thing from probably 10 versions ago. since this is an entirely different extension than the original one, it shouldn't be needed anymore.